loadtest/util/resource-helper.ts

```diff
@@ -161,7 +161,7 @@ export function deleteTestUsers(ids: Array<string>) {
     batch(
       ids.map((id) => ({
         method: "DELETE",
-        url: `${getBackendUrl()}personen/${id}`,
+        url: url`${getBackendUrl()}personen/${id}`,
```

Comment:
Ist die url, die Webseite Root? Falls ja, liefert getBackendUrl() nur den Subpath in der URL oder die gesammte URL?